elk/extraction/prompt_dataset.py

```diff
@@ -109,15 +119,16 @@ def __init__(
         # instantiations of PromptDataset (unless you set the seed to something else).
         # This allows you to just set split="train" and split="test" for any dataset
         # and not worry about train-test leakage.
-        split_name, *others = ds_dict.keys()
-        if not others:
-            print("Creating a 75/25 train-test split...")
 
-            # Don't shuffle now because we're going to shuffle later
-            ds_dict = ds_dict[split_name].train_test_split(
-                seed=cfg.seed, shuffle=False, stratify_by_column=cfg.label_column
-            )
-            assert isinstance(ds_dict, DatasetDict)
+        # split_name, *others = ds_dict.keys()
+        # if not others:
+        #     print("Creating a 75/25 train-test split...")
+
+        #     # Don't shuffle now because we're going to shuffle later
+        #     ds_dict = ds_dict[split_name].train_test_split(
+        #         seed=cfg.seed, shuffle=False, stratify_by_column=cfg.label_column
+        #     )
+        #     assert isinstance(ds_dict, DatasetDict)
```

Comment:
This comment should at least be removed? And ideally we should retain this functionality unless there's a strong reason to get rid of it— for example, `counterfact_tracing` doesn't have a train-test split by default so it's currently not clear how we'd support that dataset without the ability to create a train-test split on the fly.